src/lib/protocols/protobuf.c

```diff
@@ -78,7 +78,7 @@ protobuf_dissect_varint(struct ndpi_packet_struct const * const packet,
   size_t i;
   *value = 0;
 
-  for (i = 0; i < 9; ++i)
+  for (i = 0; i < 10; ++i)
```

Comment:
Did the protobuf dissector cause false-positives?

Reply:
@utoni , I saw you approved this PR. Could you take also a look at the above comment, please?

Reply:
Strange, looking at the code snipped it should work as you described..

Reply:
`if (i == 10)` can be removed, I think that I just forgot to remove it in an earlier iteration

Reply:
Thanks. In the last push, I restored the original loop and remove the "if" check